force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -94,7 +120,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the input is "Empty", "Null", or "Contains Text".
      */
     public static String checkString(String a) {
-        return null; // Replace null with the variable you used to store the result
+        String returnString3 = 'Contains Text';
+        if (a == '') {
+            returnString3 = 'Empty';
+        } else if (a == null) {
+            returnString3 = 'Null';
+        }
+        return returnString3;
```

Comment:
The variable names here I would be careful with. We are doing some very abstract problems here, but you should still think about what you're naming them. In the absence of context, sometimes `result` or `status` might be valid names for the variable on line 123.